src/fides/config/consent_settings.py

```diff
@@ -15,6 +15,11 @@ class ConsentSettings(FidesSettings):
         default=False, description="Toggle whether Google AC Mode is enabled."
     )
 
+    override_vendor_purposes: bool = Field(
+        default=False,
+        description="Whether or not vendor purposes can be globally overridden.",
+    )
```

Comment:
Added a temporary config setting here because I needed it in my query although this overlaps with @TheAndrewJackson's work. Happy to update to match what he's doing if he needs to go a different direction -

Reply:
Correct yes, I thought you said that Andrew was doing that as part of https://ethyca.atlassian.net/browse/PROD-1414?